tests/cases/compiler/commentsOnJSXExpressionsArePreserved.tsx

```diff
@@ -0,0 +1,15 @@
+// @module: system,commonjs
+// @jsx: react,react-jsx,react-jsxdev,preserve
+// file is interntionally not a module - this tests for a crash in the module/system transforms alongside the `react-jsx` and `react-jsxdev` outputs

```

Comment:
```suggestion
// file is intentionally not a module - this tests for a crash in the module/system transforms alongside the `react-jsx` and `react-jsxdev` outputs
```

You don't *have* to fix this typo, but there will 100% be a PR for it next October if you don't :laughing: